src/style.css

```diff
@@ -1,3 +1,69 @@
+* {
+  margin: 0;
```

Comment:
Don't use * selector for zeroing margins or paddings, you can reset it for body and other tags you are using